chromium_src/chrome/test/sources.gni

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+brave_chromium_src_chrome_test_test_support_sources = [
+ "//brave/test/base/test_brave_browser_process.cc",
+ "//brave/test/base/test_brave_browser_process.h",
+]
```

Comment:
it's better to add deps here and include them in the patch.
also add a newline at the end of the file. currently there is no newline.

Reply:
fixed